### ISSUE TITLE
Support pipeline include option in unocss nuxt module

### ISSUE THREAD

**Issue by the94air** [unocss/unocss#3242]
### Clear and concise description of the problem

Currently the unocss nuxt module support adding pipeline exclude settings although it doesn't allow us to modify the include settings to add more directories to vite build. So adding the `include` option does nothing.

### Suggested solution

Add logic to attach the `include` options similar to how its implemented with exclude option https://github.com/unocss/unocss/blob/main/packages/nuxt/src/options.ts#L33

### Additional context
Could be similar to #3232 but looks related to webpack and `node_modules` instead. Currently, the `// @unocss-include` comment method is working fine and could be considered a workaround, although its not practical with multiple files and nested folders and relying on path instead is more useful.

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/unocss/unocss/blob/main/CONTRIBUTING.md).
- [X] Read the `README.md` of using the package.
- [X] Already used the [Interactive Docs](https://unocss.dev/interactive/).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by the94air** [comment]
I can't reproduce this issue anymore which means that it was on my end and not actually a bug. Setting the options from the nuxt side seems to be more efficient. Using the uno config require manually restarting the dev script for settings to take effect. Although the `nuxt.config.ts` is instant and straightforward since nuxt will auto restart it self.

**Comment by the94air** [comment]
I was lucky to find this issue that I opened a long time ago. I came across this issue in the nuxt module again. `// @unocss-include` is the best workaround again. But I created a small reproduction. https://github.com/the94air/unocss-issues

I usually need to include some styles in my composables so its necessary that I change the content configurations. While this option is working as intended in vite (it doesn't promise working elsewhere other than with vite or webpack). It would be much nice to see it working with nuxt.
